pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -73,6 +74,16 @@ import (
 //       200: AlertGroups
 //       400: ValidationError
 
+// swagger:route POST /api/alertmanager/{Recipient}/config/api/v1/receivers/test alertmanager RoutePostTestReceivers
```

Comment:
I get the following error when load the generated spec:
```
Semantic error at paths./api/alertmanager/{Recipient}/config/api/v1/receivers/test
Declared path parameter "Recipient" needs to be defined as a path parameter at either the path or operation level
Jump to line 3488
```

You need to add `RoutePostTestReceivers` [here](https://github.com/grafana/grafana/blob/87232ea89595617760dbd95157f45fdf7e54be77/pkg/services/ngalert/api/tooling/definitions/alertmanager.go#L324).

If docker image does not work in your system can you try [installing goswagger ](https://goswagger.io/install.html)and generating the spec using instead:
`swagger generate spec -m -w definitions -o spec.json`